sources/cow_protocol/arbitrum/cow_protocol_arbitrum_sources.yml

```diff
@@ -71,6 +103,32 @@ sources:
             name: interactions
             description: "An array of internal contract interactions"
           - &tokens
+      - name: GPv2Settlement_evt_Settlement
```

Comment:
We may insert this source at the wrong position, as `&tokens` definition is interrupted by this source.